Plugin/OrderPlugin.php

```diff
@@ -48,6 +63,11 @@ public function beforeSetState(Order $subject, $state)
         if (!$subject->getPayment() || $subject->getPayment()->getMethod() != \Bolt\Boltpay\Model\Payment::METHOD_CODE) {
             return [$state];
         }
+
+        if ($subject->getIsRechargedOrder()){
```

Comment:
Could you add a comment while we need this?

Reply:
@vitaliyreznikov I've updated